veg2hab/bronnen.py

```diff
@@ -218,3 +218,21 @@ def for_geometry(self, other_gdf: gpd.GeoDataFrame) -> gpd.GeoSeries:
         """
         assert "geometry" in other_gdf.columns
         return sjoin_largest_overlap(other_gdf, self.gdf, "bodem")
+
+
+class OudeBossenkaart:
+    def __init__(self, path: Path, mask: Optional[gpd.GeoDataFrame] = None):
```

Comment:
`from_file` en evt. `from_github` toevoegen? Hoe groot is deze kaart, kunnen we 'm mee packagen?